app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/attachments/modals/attachmentImageModal.jsx

```diff
@@ -25,45 +25,93 @@ export class AttachmentImageModal extends Component {
     rotationAmount: 0,
   };
 
+  getImageSize = () =>
+    this.imageRef && this.imageRef.current
+      ? {
+          height: this.imageRef.current.clientHeight,
+          width: this.imageRef.current.clientWidth,
+        }
+      : null;
+
+  setNativeImageSize = () => {
```

Comment:
`initNativeImageSize` as it doesn't have arguments

Reply:
Fixed